Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -121,9 +121,9 @@ static void configureLayerAsGuard(CALayer *layer, NSString *groupName)
     return [layer valueForKey:interactionRegionGroupNameKey];
 }
 
-static NSString* interactionRegionGroupNameForRegion(const WebCore::InteractionRegion& interactionRegion)
+static NSString* interactionRegionGroupNameForRegion(CALayer *layer, const WebCore::InteractionRegion& interactionRegion)
 {
-    return makeString("WKInteractionRegion-"_s, interactionRegion.elementIdentifier.toUInt64());
+    return makeString("WKInteractionRegion-"_s, [layer hash], interactionRegion.elementIdentifier.toUInt64());
```

Comment:
Layers have a `PlatformLayerIdentifier` which we use in various places. I suggest using it instead (you can get it from the RemoteLayerTreeNode or via `layerID()` (slower).

Reply:
I know it’s a long shot, but is there a way to make them deterministic in the context of a test?